### PR TITLE
Add protection in RADeployer to prevent from loading rar resource in …

### PR DESCRIPTION
**Pull request by tcollignon**
…jar file

See #558 

I think it can be cherry-pick to 1.3 branch

For 2.0 branch it may another correction right?

### CODE REVIEW

**Comment by maeste** [comment]
Squash commits please and rebase

File: deployers/src/main/java/org/jboss/jca/deployers/fungal/RADeployer.java [hunk 1]
```diff
@@ -215,6 +215,24 @@ public int getOrder()
...
       }
    }
+   
+   /**
+    * Get File object of URL who locate a rar resource
+    * @param url
+    * @return a File object pointing of rar resources
+    */
+   protected File getFileFromUrl(URL url) throws URISyntaxException, IOException{
+     return new File(url.toURI());
```
Comment (maeste) [inline deployers/src/main/java/org/jboss/jca/deployers/fungal/RADeployer.java]: are you adding this method to be able to extend it in your own class?

Comment (tcollignon) [inline deployers/src/main/java/org/jboss/jca/deployers/fungal/RADeployer.java]: Yes it is, in my case I search in Jar (like #535 )
I would add another one to clean tmp file like you said

**Comment by tcollignon** [comment]
Done, now I have all my needs for my custom implem in this PR
If you want to see that tell me
Thank you

**Comment by maeste** [comment]
Merged, and pushed also on 1.3
Thanks again.

**Comment by tcollignon** [comment]
Wow very fast !
Thanks you

**Pull request merged**